### PR TITLE
feat: allow lower case for mutations

### PR DESCRIPTION
**Pull request by JonasKellerer**

### CODE REVIEW

**Review by fengelniederhammer — COMMENTED** [review]
* Should insertions be case insensitive, too?
* For consistency, we should probably implement the same thing in the variant queries, too.

**Comment by JonasKellerer** [comment]
There are now tickets for the insertions, and the variant queries: #508 #509

File: lapis2/src/main/kotlin/org/genspectrum/lapis/request/AminoAcidMutation.kt [hunk 1]
```diff
@@ -26,15 +26,15 @@ data class AminoAcidMutation(val gene: String, val position: Int, val symbol: St
             return AminoAcidMutation(
                 gene,
                 position,
-                matchGroups["symbolTo"]?.value,
+                matchGroups["symbolTo"]?.value?.toString()?.uppercase(),
```
Comment (fengelniederhammer) [inline lapis2/src/main/kotlin/org/genspectrum/lapis/request/AminoAcidMutation.kt:29]: IntelliJ claims that the `toString` is redundant
```suggestion
                matchGroups["symbolTo"]?.value?.uppercase(),
```

Comment (fengelniederhammer) [reply lapis2/src/main/kotlin/org/genspectrum/lapis/request/AminoAcidMutation.kt:29]: Same for NucleotideMutation

**Review by fengelniederhammer — APPROVED** [review]

**Pull request merged**